src/blueprint/blueprint.rs

```diff
@@ -17,15 +19,19 @@ use crate::lambda::{Expression, Lambda};
 /// It can only be generated from a valid Config.
 /// It allows us to choose a different GraphQL Backend, without re-writing all orchestration logic.
 /// It's not optimized for REST APIs (yet).
-#[derive(Clone, Debug, Default, Setters)]
+#[derive(Clone, Debug, Default, Setters, Serialize)]
```

Comment:
Can add Deserialize also so that we can start the server by pointing directly to a blueprint file.